install/README.rst

```diff
@@ -1,8 +1,8 @@
 Requirements
 ============
 
-* You need docker installed. Docker-compose is recommended in the configuration below.
```

Comment:
as docker compose is recommended, it's better to say it is required by the default configuration provided